tests/test_uninstall.py

```diff
@@ -22,14 +28,21 @@ def test_uninstall(pipx_temp_env):
     assert not run_pipx_cli(["uninstall", "pycowsay"])
 
 
+def test_uninstall_global(pipx_temp_env):
+    if sys.platform.startswith("win"):
+        pytest.skip("This behavior is undefined on Windows")
```

Comment:
There are other tests that could be modified similarly.

```suggestion
@pytest.mark.skipif(sys.platform.startswith("win"), reason="This behavior is undefined on Windows")
def test_uninstall_global(pipx_temp_env):
```

Reply:
Big + on this. Way easier to read. Could also turn this skip reason into a shareable decorator: https://docs.pytest.org/en/latest/how-to/skipping.html#id1